<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hi</class>
 <widget class="QMainWindow" name="hi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flowmeter + modbus reader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="arduinobox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>227</width>
      <height>72</height>
     </rect>
    </property>
    <property name="title">
     <string>Arduino</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>33</y>
       <width>29</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QComboBox" name="arduino_port">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>33</y>
       <width>92</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="ard_portrefresh">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>33</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>refresh icon.png</normaloff>refresh icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="modbusbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>227</width>
      <height>174</height>
     </rect>
    </property>
    <property name="title">
     <string>Modbus</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>33</y>
       <width>29</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QComboBox" name="modbus_port">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>33</y>
       <width>92</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>67</y>
       <width>94</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Slave address</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="slave_address">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>67</y>
       <width>92</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>247</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>101</y>
       <width>88</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Inlet register</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="inlet_register">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>101</y>
       <width>92</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>18182</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>135</y>
       <width>98</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Outlet register</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="outlet_register">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>135</y>
       <width>92</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>18184</number>
     </property>
    </widget>
    <widget class="QToolButton" name="mod_portrefresh">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>33</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>refresh icon.png</normaloff>refresh icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>33</y>
       <width>52</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Log file</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file_path_box">
     <property name="geometry">
      <rect>
       <x>88</x>
       <y>33</y>
       <width>251</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="run_number">
     <property name="geometry">
      <rect>
       <x>88</x>
       <y>70</y>
       <width>54</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>70</y>
       <width>65</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Run num</string>
     </property>
    </widget>
    <widget class="QToolButton" name="filebrowse">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>33</y>
       <width>32</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startstop_button">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="statusdisplay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>145</y>
      <width>381</width>
      <height>116</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
